brave/brave-ios
# Fix #7880: Coinbase Pay on ramp

## Summary of Changes
- Integrates Coinbase Pay on ramp into supported buy providers screen

This pull request fixes #7880

## Submitter Checklist:


## Test Plan:
1. Open Buy screen
2. Select GBP, CAD, EUR, or USD as currency
3. Optionally enter a value, then tap `Select purchase method`
4. Verify Coinbase is displayed as on-ramp provider
5. Verify `Buy with Coinbase Pay` opens expected Coinbase Pay URL in browser.


## Screenshots:

![Coinbase Pay on ramp](https://github.com/brave/brave-ios/assets/5314553/46e66a67-4192-47e2-b41c-21174ee08a90)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
